### PR TITLE
bin/mh: replace stray "print" with "print_log".

### PR DESCRIPTION
**Pull request by bpmartin20**
Looks like this print should be a print_log. All the others messages in this routine use print_log, so I think this one just got overlooked.

### CODE REVIEW

File: bin/mh [hunk 1]
```diff
@@ -4740,7 +4740,7 @@ sub process_external_command {
     $cmd =~ s/^\s+//;                            # Deletes leading blanks
     $cmd =~ s/\s+$//;                            # Deletes trailing blanks
 
-    print "Running external command: $cmd set by $set_by\n"
+    print_log "Running external command: $cmd set by $set_by\n"
```
Comment (marcmerlin) [inline bin/mh:4743]: yes to this, thanks

File: lib/read_table_A.pl [hunk 2]
```diff
@@ -2077,18 +2124,16 @@ sub read_table_A {
...
+                $code .= sprintf "\$%-35s -> add(\$%s);\n", $group, $name;
+                $groups{$group}{$name}++;
+            }
         }
     }
 
-    if ($additional_code) {
-        $code .= $additional_code;
-    }
-
```
Comment (marcmerlin) [inline lib/read_table_A.pl:2091]: I didn't read the whole diff carefully, but I believe you may be running this code at a different time now, and might generate code that has a different order?

**Review by marcmerlin — COMMENTED** [review]
you mixed 2 commits into one review, please remove the 2nd one which doesn't match the title of the PR.
I'm fine approving the first one right now, the 2nd one is a bigger change that someone will have to review very carefluly since clearly this change will affect every single user, and it's not trivial to tell that it's ok or safe for everyone just by staring at at it

File: lib/read_table_A.pl (hunk 2 above)
Comment (bpmartin20) [reply lib/read_table_A.pl:2091]: Thanks. Will cancel this PR if I have the authority. I'll resubmit, regardless.

**Comment by bpmartin20** [comment]
Will resubmit.

**Pull request closed without merging**